src/contracts/token.ts

```diff
@@ -270,11 +268,6 @@ export class Token extends Erc20<TokenERC20> {
     holder: string,
```

Comment:
let's get rid of this function, it's just confusing and has almost no value

Reply:
Feels odd to remove this (assuming you're referring to `burnFrom` because its the only function on the token interface that calls the `burnFrom` function on the contract: https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/ad67d48f0914a15a9350ef42672322058b48d5b1/contracts/token/ERC20/extensions/ERC20BurnableUpgradeable.sol#L41

Also, it's existed on our token contract for a long time now. We should probably have contract interface for the functions on the contract right? Otherwise just shouldn't be on the contract if it really has no use (but since OZ has it, maybe people use it for something).

Reply:
Will delete anyway for now